### ISSUE TITLE
Team check-in

### ISSUE THREAD

**Issue by TeriDibbits** [paprika-spice/web-app#52]

**Comment by TeriDibbits** [comment]
My tasks included creating and collecting any assets for our app. This included creating icons, finding appropriate photography, and preparing written content that could possibly be on each page. In terms of coding my tasks were to create the matches page that displayed a list of cards for each available room. I also created a second page for room booking; this page included all the rooms info, a photo of the room, and a booking button. Lastly, I reviewed my team members pull requests. I looked over their code to find any fixes or adjustments to be made and requested changes.

**Comment by scallsen** [comment]
I created the style tile, which provided assets for my other team members to create all other pages on our site. I now need to complete the home page in class, and our site should have a complete flow.

**Comment by MJholland** [comment]
This week I implemented Jeykll and created the default layout (header & footer). My other task was to create the first booking step page. This involved creating a form with a date picker field and two drop downs with select times. Originally I used the input type time but the picker that is supplied by the browser isn't the best and styling our own would require javascript/jquery/bootstrap/some kind of library. The date picker supplied by the browser isn't as bad as the time picker but because of the need of library I couldn't change the format from YYYY/MM/DD to anything else. I also reviewed Teri's pull requests and asked for changes which she made.
